integration-tests/create-react-app/.gitignore

```diff
@@ -0,0 +1,21 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
All the files in this `create-react-app` folder are exactly what you get from generating a new project with `create-react-app`. All I did was install `react-percy` and add a percy snapshot file.